### ISSUE TITLE
Any chance of adding English as language in settings of the app?

### ISSUE THREAD

**Issue by Tor101** [koishijs/koishi-desktop#108]
Any chance of adding English as language in settings of the app?

**Comment by shigma** [comment]
|  | I18n Functionallity | English Translations |
| -- |:--:|:--:|
| Koishi | ✅ | ✅ |
| Koishi Desktop | ✅ | ✅ |
| Koishi WebUI | 4.13 | 4.13 ~ 4.14 |

**Comment by Tor101** [comment]
I mange to figure out that there is a local link too where I can open in browser and translate with native browser translator.
Thank you.

**Comment by MaikoTan** [comment]
Yep, I would do the translating from Chinese to English / Japanese as soon as possible when Shigma added the internationalisation support for the Koishi WebUI, also some of the plugins that are made by the Koishi Team.
But other Koishi plugins may still have no localisations, which is up to the plugin authors.
